pkg/flare/flare.go

```diff
@@ -0,0 +1,199 @@
+// Package flare is used for collecting, sanitizing, and packaging lakeFS configuration, log files, and environment variables
+// for debugging and troubleshooting purposes.
+package flare
+
+import (
+	"crypto/sha512"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/octarinesec/secret-detector/pkg/scanner"
+	"github.com/octarinesec/secret-detector/pkg/secrets"
+	"gopkg.in/yaml.v3"
+)
+
+var defaultEnvVarPrefixes = []string{"LAKEFS_", "HTTP_", "HOSTNAME"}
```

Comment:
This list filters which env vars are processed. Since `AWS_` isn't on the list, we won't write it to file.

Reply:
I understand.  I **really** need to know whether the user has `AWS_*` variables set, otherwise it can be hard to understand what is going on whether the user complains.  (I don't particularly care about their values, I do want to know.)  Probably also for other potential default envariables, e.g. AFAIU our PostgreSQL driver pgx will end up parsing a tonne of variables `PG*` (no underscore), and I want to know at least which ones are set.

Reply:
@arielshaqed So you're saying we should add a prefixs `AWS_` and `PG`?